core/src/main/scala/org/apache/spark/ui/PagedTable.scala

```diff
@@ -95,7 +106,12 @@ private[ui] trait PagedTable[T] {
         val PageData(totalPages, _) = _dataSource.pageData(1)
         <div>
           {pageNavigation(1, _dataSource.pageSize, totalPages)}
-          <div class="alert alert-error">{e.getMessage}</div>
+          <div class="alert alert-error">
+            <p>Error while rendering table:</p>
+            <pre>
+              {Utils.exceptionString(e)}
```

Comment:
This is just for `IndexOutOfBoundsException`.  Using `e.getMessage` would be enough. `Utils.exceptionString` is really not friendly, e.g.,

<img width="1347" alt="screen shot 2015-12-22 at 2 21 14 pm" src="https://cloud.githubusercontent.com/assets/1000778/11967403/615c6b9e-a8b7-11e5-800c-1130bb7ec7c5.png">